code/cryptography/src/autokey_cipher/autokey_cipher.cpp

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <string.h>
+
+void flush_stdin()
```

Comment:
This function isn't used in this file, so just remove it. Also include the `<string.h>` header.